sql/dbt/uniswap_v2/models/analytics/uniswap_v2_ethereum__token_prices.sql

```diff
@@ -0,0 +1,26 @@
+{% set swaps = ref("uniswap_v2_ethereum__swaps") %}
+
+with final as (
+    select 
+        transaction_hash
+        , log_index
+        , amount_in
+        , amount_out
+        , token_in__id
+        , case 
+            when token_in__id = '0xdAC17F958D2ee523a2206206994597C13D831ec7' then 6 else 18
```

Comment:
This is very fragile. What's the plan of handling this when we have dynamic data sources?

Reply:
Right now, I have hardcoded the token addresses and token decimals. In the future, we can also pull token-related information such as decimals, symbols, names, etc. with the help of dynamic data sources.